test-db-container/README.md

```diff
@@ -0,0 +1,49 @@
+# Local SQL Server Docker Container for Testing
+
+This repository provides a Docker container with a pre-configured SQL Server instance that contains a sample database schema and initial data.
+
+## Getting Started
+
+1. **Build the Docker Image:**
+
+    Build the Docker image using the provided Dockerfile and initialization script.
+
+    ```bash
+    docker build -t test-db-container .
+    ```
+
+2. **Run the Docker Container:**
+
+    Start a Docker container from the built image. Replace `YourStrongPassword123` with a strong password for the SQL Server 'sa' user.
```

Comment:
There is no such string in your PR 'YourStrongPassword123'

Reply:
I forgot to change it; I will fix it.